### PR TITLE
Auth tests

### PR DESCRIPTION
**Pull request by Drapegnik**
* Add auth api testing with [`supertest`]()
* Add [`TravisCI`](https://travis-ci.org/babajka/babajka-backend) integration
  * run tests
  * run `eslint`
  * build
* Add [`Coveralls`](https://coveralls.io/github/babajka/babajka-backend) integration
  * count test coverage
for correct `coverage` script work you should add `COVERALLS_REPO_TOKEN` env var, check it on [`GDrive`](https://drive.google.com/drive/u/0/folders/0B_73TCUw02_JTTQwaTUyd0FUZVU)

### CODE REVIEW

**Comment by Drapegnik** [comment]
[improve backend testing](https://trello.com/c/MK0z6JPr/49-improve-backend-testing)

**Comment by UladBohdan** [comment]
I have one test failing:

```1) User should save user with password:```
```Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.```

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - "7"
+services:
+  - mongodb
+before_script: npm run init-db
+script:
+  - npm run lint
+  - npm run build
+  - npm test
+notifications:
+  slack: babajka:Taq9qEu87dDfrZv6D7SP360J
```
Comment (UladBohdan) [inline .travis.yml]: Is that our slack token? Is it safe to store it here?

Comment (Drapegnik) [inline .travis.yml]: > Your integration token is semi-secret

Comment (UladBohdan) [inline .travis.yml]: What does that mean?

**Comment by UladBohdan** [comment]
Slack issue resolved.
(changes on file formatting were made by travis CLI so that must remain)

**Pull request merged**